app/assets/javascripts/discourse/app/components/copy-button.js

```diff
@@ -0,0 +1,16 @@
+import Component from "@ember/component";
+import { action } from "@ember/object";
+import copyText from "discourse/lib/copy-text";
+
+export default Component.extend({
+  tagName: "",
+
+  @action
+  copy() {
+    const $copyRange = $('<p id="copy-range"></p>');
```

Comment:
This not especially needed in this PR but if we could get rid of jquery here that would be awesome

Reply:
Got rid of jquery and reused existing `<input>` element. :heavy_check_mark: :heavy_check_mark: